creator/ingest_runs/models.py

```diff
@@ -77,6 +85,35 @@ class Meta:
         help_text="The user who submitted this ingest run",
     )
 
+    @transition(field=state, source="waiting", target="running")
+    def started(self):
+        """
+        Begin running the ingest process.
+        """
+        return
+
+    @transition(field=state, source="running", target="complete")
+    def complete(self):
+        """
+        Finish running the ingest process without error.
+        """
+        return
+
+    @transition(field=state, source="running", target="canceled")
+    def canceled(self):
```

Comment:
@Gio Similar to `started` you may want to delete the `cancel` placeholder and just call this method `cancel`